extensions/amp-analytics/0.1/events.js

```diff
@@ -1279,19 +1293,32 @@ export class VideoEventTracker extends EventTracker {
   /** @override */
   add(context, eventType, config, listener) {
     const videoSpec = config['videoSpec'] || {};
-    const selector = config['selector'] || videoSpec['selector'];
-    const selectionMethod = config['selectionMethod'] || null;
-    const targetReady = this.root.getElement(
-      context,
-      selector,
-      selectionMethod
+    const selectorValue = userAssert(
+      config['selector'] || videoSpec['selector'],
+      'Missing required selector on video trigger'
+    );
+    const selectorArray = isArray(selectorValue)
+      ? selectorValue
+      : [selectorValue];
+
+    userAssert(
+      selectorArray.length,
+      'Missing required selector on video trigger'
```

Comment:
This works on both a string and an array:

```suggestion
    userAssert(
      selector.length,
      'Missing required selector on video trigger'
```